### PR TITLE
Added Candlestick chart.

### PR DESCRIPTION
**Pull request by fanij**
## PR Templates

**jira link:**

**Did you tested this feature before makis this PR?**
- [ ]
**Is Units test written for this?**
- [ ]
**Is UI tests written for this?**

## Deployment Changes
- [ ] Deployment Changes

## UI Changes
- Added CandleStick chart.

candlestick01
candlestick02

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            animationSelected = .linear
+        case 1:
+            animationSelected = .easeInCubic
+        default:
+            animationSelected = .easeOutCubic
+        }
+        hideChartData = true
+    }
+    
+    func updateChartData()  {
```
Comment (randhirkumar65) [inline ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Mark this method as private

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        if hideChartData {
+            hideChartData = false
+            candleStickChartView.data = nil
+            return
+        }
+    }
+    
+    func setCandleChartData() {
```
Comment (randhirkumar65) [inline ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Mark this method as private

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        candleStickChartView.autoScaleMinMaxEnabled = true
+        setCandleDataCount(8, range: 30)
+    }
+    
+    func setCandleDataCount(_ count: Int, range: UInt32) {
```
Comment (randhirkumar65) [inline ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Here too , can you please mark this private func

**Review by randhirkumar65 — CHANGES_REQUESTED** [review]
@fanij Thanks for the contribution , overall changes look good to me, left a very minor comments

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift (hunk 1 above)
Comment (fanij) [reply ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Marked as private.

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift (hunk 2 above)
Comment (fanij) [reply ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Marked as private.

File: ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift (hunk 3 above)
Comment (fanij) [reply ChartDemo/Features/CandleStickChart/ViewController/CandleStickChartViewController.swift]: Marked as private.

**Review by randhirkumar65 — APPROVED** [review]
Looks good to me, Thanks !!!

**Pull request merged**